spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -111,6 +111,7 @@
 "CDDL or GPLv2 with exceptions": CDDL-1.0 OR GPL-2.0-only WITH Classpath-exception-2.0
 "CDDL v1.0 / GPL v2 dual license": CDDL-1.0 OR GPL-2.0-only
 "CDDL v1.1 / GPL v2 dual license": CDDL-1.0 OR GPL-2.0-only
+"CDDL-1.1 OR GPL 2.0 only WITH Classpath exception 2.0": CDDL-1.1 OR GPL-2.0-only WITH Classpath-exception-2.0
```

Comment:
I am unable to find this string in the pom file mentioned in the commit message.